derive-error-chain/src/lib.rs

```diff
@@ -640,14 +671,15 @@ This struct is made of three things:
 			", error_kind_name);
 
 			let result_wrapper = result_name.map(|result_name| quote! {
+				#[allow(non_camel_case_types)]
 				/// Convenient wrapper around `::std::result::Result`
-				pub type #result_name<T> = ::std::result::Result<T, #error_name>;
+				pub type #result_name#ty_generics_ext1 = ::std::result::Result<DERIVE_ERROR_CHAIN_T, #error_name#ty_generics>;
```

Comment:
Space between the tokens. `#result_name #ty_generics`, `#error_name #ty_generics`, etc. There are some more lines that also need it.